src/styles/states.css

```diff
@@ -4,56 +4,91 @@
   dark:bg-state-completed-600
   dark:text-state-completed-50
 }
+.state-text--completed { @apply
+  text-state-completed-50
+  dark:text-state-completed-600
+}
```

Comment:
I don't super love this CSS file. But I have intent to create a raft around updating state colors more broadly, so I know it's something that will be revisited.